tools/bias_lang_migration.sh

```diff
@@ -0,0 +1,678 @@
+#!/bin/bash
+#description  :This script migrates existing CRs deployed to non-Bias-Language replacements.
+#version      :1.0
+
```

Comment:
Team, whenever you get a chance, please let me know if you have any comments on this script. 
@akondur , @kashok-splunk , @gaurav-splunk , @jryb , @kumarajeet. 
Thank you!